js/sider/OppfolgingsplanSide.js

```diff
@@ -301,7 +300,9 @@ export function mapStateToProps(state, ownProps) {
         && sykmeldinger.hentet;
     const erSykmeldteHentet = state.sykmeldte.hentet && !state.sykmeldte.hentingFeilet;
     const skalHenteBerikelse = beregnSkalHenteSykmeldtBerikelse(sykmeldt, state);
-
+    const sykmeldtPerson = state.person.data && state.person.data.filter((s) => {
```

Comment:
`person.data`- feltet blir initialisert som et tomt array og tomme arrays evalueres til truthy. Men hvis vi har lyst å sjekke om feltene eksisterer kan vi eventuelt bruke optional chaining. 

Nitpick: Vi kan bruke `.find` for å finne det første elementet i listen med kriteriene. Den vil også returnere `undefined` for tomme lister eller ingen elementer funnet.

Gjelder også for endringene i `js/sider/OppfolgingsplanerSide.js`

Reply:
Vet ikke helt hvorfor jeg får e-post watch om dette repoet - jeg bør sikkert unsubscribe 😄  - men, et tips kan være lage en custom hook for sykmeldt person. Så kan du hente den med f.eks useSykmeldtPerson(), evnt useSykmeldtPerson(fnr)

Reply:
Det er opp til deg men vi er veldig taknemlige for tips og innspill :D 

Godt inn spill! Det burde vi vurdere. Med custom hook tenker du da en hjelpe funksjon som henter sykemeldt person i mapperne (er relativ ny i React)? For å hente `store` i funksjoner som ikke blir composed av `connect`, er det bare å importere `store` ?

@AudunSorheim

Reply:
Med hooks så slipper man å bruke connected i det hele tatt, det blir en del mindre styring med parameter osv.

Dersom du oppretter f.eks en "hooks" mappe i repoet deres, med for eksempel en fil sykmeldtHooks.js, så kan du lage dine egne custom hooks der for å hente ut og transformere data tilknyttet sykmeldte.

f.eks noe ala
```
import { useSelector } from 'react';

export function useSykmeldtPerson(fnr) {
  const data = useSelector(state => state.person.data) || {};

  return data.filter((s) => {
        return `${s.fnr}` === sykmeldt.fnr;
    })[0];
}
```

Da kan du hver gang du trenger å hente ut sykmeldtPerson skrive
```
const sykmeldtPerson = useSykmeldtPerson(etEllerAnnetFnr)
```

..evnt om du kan utlede fnr fra redux staten, så slipper du å sende den inn som parameter. Men kjenner ikke til deres applikasjon.. :)

https://reactjs.org/docs/hooks-intro.html